tools/Dockerfile

```diff
@@ -81,23 +58,39 @@ RUN cp -p LICENSE /usr/share/licenses/eksctl && \
       --out-dir /usr/share/licenses/eksctl/vendor \
       go-vendor ./vendor
 RUN curl -L "${EKSCTL_BINARY_URL}" \
-      -o eksctl_${EKSCTL_VERSION}_${GOOS}_${GOARCH}.tar.gz && \
-    grep eksctl_${EKSCTL_VERSION}_${GOOS}_${GOARCH}.tar.gz \
+      -o "eksctl_${EKSCTL_VERSION}_${GOOS}_${GOARCH}.tar.gz" && \
+    grep "eksctl_${EKSCTL_VERSION}_${GOOS}_${GOARCH}.tar.gz" \
       /hashes/eksctl | sha512sum --check - && \
-    tar -xf eksctl_${EKSCTL_VERSION}_${GOOS}_${GOARCH}.tar.gz -C /tmp && \
-    rm eksctl_${EKSCTL_VERSION}_${GOOS}_${GOARCH}.tar.gz
+    tar -xf "eksctl_${EKSCTL_VERSION}_${GOOS}_${GOARCH}.tar.gz" && \
+    rm "eksctl_${EKSCTL_VERSION}_${GOOS}_${GOARCH}.tar.gz"
+
+# =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^=
+FROM build-go as eksctl-anywhere-build
+
+USER root
+RUN mkdir -p /usr/share/licenses/eksctl-anywhere && \
+    chown -R builder:builder /usr/share/licenses/eksctl-anywhere
+
+ARG EKSA_VERSION=0.12.2
+ARG EKSA_SOURCE_URL="https://github.com/aws/eks-anywhere/archive/refs/tags/v${EKSA_VERSION}.tar.gz"
+
+ARG EKSA_RELEASE_NUMBER=24
+# TODO EKS-A currently does not provide an ARM64 binary, add the hash for it when they do
+ARG EKSA_BINARY_URL="https://anywhere-assets.eks.amazonaws.com/releases/eks-a/${EKSA_RELEASE_NUMBER}/artifacts/eks-a/v${EKSA_VERSION}/${GOOS}/amd64/eksctl-anywhere-v${EKSA_VERSION}-${GOOS}-amd64.tar.gz"
 
-# eksctl-anywhere
 # TODO Try and build this; it's not straightforward with the tight coupling between EKS-A releases and its bundles
+USER builder
 WORKDIR /home/builder/
-RUN mkdir eksctl-anywhere && curl -L ${EKSA_SOURCE_URL} \
-      -o eksctl_anywhere_${EKSA_VERSION}.tar.gz && \
-    grep eksctl_anywhere_${EKSA_VERSION}.tar.gz \
-      /hashes/eksctl | sha512sum --check - && \
-    tar -xf eksctl_anywhere_${EKSA_VERSION}.tar.gz --strip-components 1 -C eksctl-anywhere && \
-    rm eksctl_anywhere_${EKSA_VERSION}.tar.gz
+RUN mkdir eksctl-anywhere && \
+    curl -L "${EKSA_SOURCE_URL}" \
+      -o "eksctl_anywhere_${EKSA_VERSION}.tar.gz" && \
+    grep "eksctl_anywhere_${EKSA_VERSION}.tar.gz" \
+      /hashes/eksctl-anywhere | sha512sum --check - && \
+    tar -xf "eksctl_anywhere_${EKSA_VERSION}.tar.gz" \
+      --strip-components 1 -C eksctl-anywhere && \
+    rm "eksctl_anywhere_${EKSA_VERSION}.tar.gz"
 
-USER root
+USER builder
```

Comment:
Set to builder already above. Doesn't look like this `USER` line is necessary now that line 82 was added.